powerline/segments/common/sys.py

```diff
@@ -13,7 +13,8 @@
 cpu_count = None
 
 
-def system_load(pl, format='{avg:.1f}', threshold_good=1, threshold_bad=2, track_cpu_count=False):
+def system_load(pl, format='{avg:.1f}', threshold_good=1, threshold_bad=2,
+		track_cpu_count=False, short=False):
```

Comment:
You should either leave everything on one line or align with first argument (note: *alignment* uses spaces).